### PR TITLE
Hackaton-01 - Add first model interation for rates.

### PR DESCRIPTION
**Pull request by anfbermudezme**
Add model and the respective backends.

### CODE REVIEW

File: eox_rates/apps_module/rating/models.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from django.db.models import CheckConstraint, Q, UniqueConstraint
+from django.contrib.auth.models import User
+from eox_rates.edxapp_wrapper.course_module import get_course_overview
+
+
+class EoxRates(models.Model):
+    """
+    Model to persist rating for a course.
+    """
+    rate = models.FloatField(validators=[MinValueValidator(0.0), MaxValueValidator(5.0)])
```
Comment (Alec4r) [inline eox_rates/apps_module/rating/models.py]: We will use INT instead of Float.

File: eox_rates/apps_module/rating/models.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from django.contrib.auth.models import User
+from eox_rates.edxapp_wrapper.course_module import get_course_overview
+
+
+class EoxRates(TimeStampedModel):
+    """
+    Model to persist rating for a course.
+    """
+    rate = models.PositiveIntegerField(default=0, validators=[MinValueValidator(1), MaxValueValidator(5)])
+    story = models.ForeignKey(Story, on_delete=models.CASCADE)
```
Comment (Alec4r) [inline eox_rates/apps_module/rating/models.py]: What is this?

Comment (anfbermudezme) [reply eox_rates/apps_module/rating/models.py]: Unneccesary field.

File: eox_rates/apps_module/rating/models.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+from django.contrib.auth.models import User
+from eox_rates.edxapp_wrapper.course_module import get_course_overview
+
+
+class EoxRates(models.Model):
+    """
+    Model to persist rating for a course.
+    """
+    rate = models.PositiveIntegerField(default=0, validators=[MinValueValidator(1), MaxValueValidator(5)])
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```
Comment (Alec4r) [inline eox_rates/apps_module/rating/models.py]: This should be a ForeingKey too, and we should use UniqueConstraint for this.

Maybe something like this:
    class Meta:
        constraints = [
            models.UniqueConstraint(
                fields=["user", "rating"], name="unique_course_user_rating"
            )
        ]

**Review by Alec4r — APPROVED** [review]

**Pull request merged**